recipe/build_osx_arm64.sh

```diff
@@ -0,0 +1,28 @@
+export VERBOSE=1
```

Comment:
Do we want compilation to always be verbose?

Reply:
It adds a moderate amount to the log size, but provides useful debugging information if the builds ever break in the future. That makes one less step to debugging a broken build.